### PR TITLE
Fixed ConfirmRegistration routing issues

### PR DESCRIPTION
**Pull request by liquidcircle**

### CODE REVIEW

**Comment by vbudilov** [comment]
Hi @liquidcircle , 

Let me review the changes. 

Thanks,
Vladimir

**Comment by liquidcircle** [comment]
Thanks Vladimir!

The repo steps are:


  *   Sign up for an account but do not confirm your account.
  *   Return to the login page and login with your new credentials.

Currently this throws an error, rather than forwarding the user to the confirm registration page.  Also note that the username (email address) wasn’t being passed correctly to the confirm registration page.

The next item on my list is to login the user after they register, rather than requiring the user to login.  I’ll check that in as well, and let you know, if you’re keen on that change.

Cheers,

Alan


On Jan 6, 2017, at 10:14 AM, Vladimir Budilov <notifications@github.com<mailto:notifications@github.com>> wrote:

Vladimir

**Review by vbudilov — APPROVED** [review]

File: src/app/public/auth/auth.component.ts [hunk 1]
```diff
@@ -48,6 +48,10 @@ export class LoginComponent implements CognitoCallback, LoggedInCallback, OnInit
         if (message != null) { //error
             this.errorMessage = message;
             console.log("result: " + this.errorMessage);
+            if (this.errorMessage === 'User is not confirmed.') {
```
Comment (vbudilov) [inline src/app/public/auth/auth.component.ts:51]: I'll merge this for now, but this is not the best check. It's definitely not going to hurt, but the actual message might change and this conditional statement will default to false.

**Comment by vbudilov** [comment]
@liquidcircle , 

Thanks for the pull request -- I merged it but had a lot of local changes so the files moved around. Make sure you pull the latest. 

Also, in regards to "The next item on my list is to login the user after they register, rather than requiring the user to login.  I’ll check that in as well, and let you know, if you’re keen on that change." -- that's a helpful feature :) Just didn't have the time to implement this myself -- btw I hope you mean "after they confirm the code", rather than "register". Correct? 

Thanks,
Vladimir

**Comment by liquidcircle** [comment]
Thanks Vladimir!

Regarding the login piece, yes, correct... after they enter their confirmation code.

Cheers,

Alan

**Pull request merged**